src/utils/relay.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace StarkInfra\Utils;
+use StarkInfra\Settings;
+use StarkCore\Utils\StarkHost;
+
+
+class Relay
+{
+    private const _apiVersion = "v2";
+    private const _sdkVersion = "v0.4.0";
```

Comment:
Era uma boa avisar o Dotts de mudar a versão quando ele for deplorado, ou alterar agora já

Reply:
Vou mudar a definição dessas constantes para o settings.php, daí fica mais exposto